dom-event-delegation/main.js

```diff
@@ -0,0 +1,16 @@
+
+var tasks = document.querySelector('.task-list');
+
+tasks.addEventListener('click', handleTasks)
+
+function handleTasks(event) {
+  console.log(event.target);
+  console.log(event.target.tagName);
+  if (event.target !== 'button') {
```

Comment:
this is not the proper conditional statement for here. it is not checking the tagName property, and the `BUTTON` needs to be capitalized. Make sure this is working properly only when you click on the button and no other element.